pkg/apis/servicecatalog/register.go

```diff
@@ -56,6 +56,13 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&metav1.GetOptions{},
 
 		&Broker{},
```

Comment:
This list below makes sense to me. So my question is more of an educational one about the list above.
Are those "api" and "metav1" types there so that a GET on a high-level URL will return things like the list of our resource types?